modin/engines/ray/utils.py

```diff
@@ -136,7 +137,15 @@ def initialize_ray(
             # In case anything failed above, we can still improve the memory for Modin.
             if object_store_memory is None:
                 # Round down to the nearest Gigabyte.
-                system_memory = ray._private.utils.get_system_memory()
+                if sys.platform == "linux" or sys.platform == "linux2":
+                    shm_fd = os.open("/dev/shm", os.O_RDONLY)
+                    try:
+                        shm_stats = os.fstatvfs(shm_fd)
+                        system_memory = shm_stats.f_bsize * shm_stats.f_bavail
```

Comment:
What if `/dev/shm` is very small (like a docker container)? Should we handle that here?

Reply:
Probably, yes. We can raise warning to user in case when size(/dev/shm) < size(RAM // 2) and use default in Ray for `object_store_memory`. In warning message we can require from the user increasing of `/dev/shm` with `--shm-size` in Docker or deleting files from `/dev/shm`